File: src/main/java/com/seetatech/seetaverify/SeetaApplication.java
package com.seetatech.seetaverify;

import android.app.Application;


public class SeetaApplication extends Application {
    private static SeetaApplication instance;

    public static SeetaApplication getInstance() {
        return instance;
    }

    @Override
    public void onCreate() {
        super.onCreate();
        instance = this;
    }


}

File: app/src/main/java/com/seetatech/demo/SeetaApplication.kt
package com.seetatech.demo

import android.app.Application
import com.seetatech.demo.SeetaApplication

class SeetaApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        instance = this
    }

    override fun onTerminate() {
        super.onTerminate()
    }

    companion object {
        var instance: SeetaApplication? = null
            private set
    }
}